inventory_service/src/item_repository.py

```diff
@@ -21,6 +21,7 @@ def update_item_repository(item_id,new_item):
         result = update_item(cursor,item_id,new_item)
         conn.commit()
         return result
+        
```

Comment:
the function to connect to database can be refactored to handle multiple connections:
![image](https://user-images.githubusercontent.com/70574489/217153259-74e21629-3a8b-462b-b543-fad613c799d7.png)